python/ray/tests/test_client_reconnect.py

```diff
@@ -367,17 +367,17 @@ def fail_every_three(_):
 
     @ray.remote
     def large_result():
-        # 1024x1024x128 float64 matrix (1024 MiB). With 64MiB chunk size,
+        # 1024x1024x6 float64 matrix (96 MiB). With 5MiB chunk size,
```

Comment:
Note for other reviewers: we're adjusting the size of the transferred object here because these tests inject faults into the gRPC stream every 3 chunks -- since this PR decreases chunk size, it increased the number of chunks/the number of faults injected, which triggered exponential backoff and slowed down the test.